src/components/BalanceDistributionBar.js

```diff
@@ -103,4 +102,20 @@ class BalanceDistributionBar { // eslint-disable-line no-unused-vars
         }
         return $bar;
     }
+
+    /**
+     * @param {number} units
+     * @param {Asset} currency
+     * @param {number} rate
+     * @returns {number}
+     */
+    _unitsToFiat(units, currency, rate) {
+        switch (currency) {
+            case 'NIM': return Nimiq.Policy.lunasToCoins(units) * rate;
+            case 'BTC': return BitcoinUtils.satoshisToCoins(units) * rate;
+            case 'USDC': return PolygonUtils.centsToCoins(units) * rate;
```

Comment:
I guess this comment follow the previous one about cents. We can keep it like that even if it's not exactly what's happenning (since it's not cents but 1/10000 of a cent iirc) or change it for consistency to something like `weiToCoins` or `usdcweiToCoins` or anything else more correct than `cents`.